package.json

```diff
@@ -29,7 +29,7 @@
   },
   "homepage": "https://github.com/za-creature/coffeescope",
   "license": "MIT",
-  "main": "lib",
+  "main": "src",
```

Comment:
This allows the module to be loaded as coffeescript instead of needing to be compiled to JS 

(the normal `npm prepublish` packaging)